File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/ContactFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.Contact;

public class ContactFilterValidator extends FilterValidator<Contact> {

    public ContactFilterValidator(CustomValidator<Contact> validator) {
        super(Contact.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/ContactFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.std.Contact
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class ContactFilterValidator(validator: Validator<Contact>) : FilterValidator<Contact>(Contact::class, validator)
